packages/frontend-plugin-api/api-report.md

```diff
@@ -1242,7 +1233,55 @@ export { FetchApi };
 export { fetchApiRef };
 
 // @public (undocumented)
-export type FrontendFeature = BackstagePlugin | ExtensionOverrides;
+export type FrontendFeature =
+  | FrontendPlugin
+  | FrontendModule
+  | ExtensionOverrides;
```

Comment:
I realize we could probably move this over to app-api instead, don't think it's actually used here?

Regarding https://github.com/backstage/backstage/pull/26245/files#r1733051780 I think we should add `{ $$type: '@backstage/BackstagePlugin' }` here too for compatibility.